runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -247,44 +248,22 @@ ExecutorFactory::createLinearExecutor(std::unique_ptr<compiler::LoweredGraph> lo
 
   initializeBackendContext(lowered_graph.get());
 
-  // linearize
-  assert(!lowered_graph->graph().isBuildingPhase());
-
-  /*************************************************
-   * Backend dependent analysis & optimization phase
-   *************************************************/
-
-  for (auto &pair : backend_contexts)
-  {
-    auto &optimizer = pair.second->optimizer;
-    if (optimizer)
-      optimizer->optimize();
-  }
-
-  /**********************************************************
-   * Backend dependent analysis & optimization phase finished
-   **********************************************************/
-
-  /***********************
-   * Code generation phase
-   ***********************/
+  TensorBuilders tensor_builders{lowered_graph->backend_contexts(), true};
```

Comment:
`ExecutorFactory` now just calls `genTensors` and `genKernels` and it does not refer any `backend::Ixxxx` classes. Removing those classes will be done in the following PRs.